routers/repo/repo.go

```diff
@@ -146,6 +146,9 @@ func CreatePost(ctx *context.Context, form auth.CreateRepoForm) {
 		if errDelete := models.DeleteRepository(ctxUser.ID, repo.ID); errDelete != nil {
 			log.Error(4, "DeleteRepository: %v", errDelete)
 		}
+		if unit := repo.Units[models.UnitTypeIssues]; unit != nil {
```

Comment:
I'm confused here. Why do we need to update the repo's unit if we just deleted the repo? It doesn't look like this write even reaches the database.